### ISSUE TITLE
404 Error ist just "Info" and does not tell the Pool/URL

### ISSUE THREAD

**Issue by Knight1** [cjdelisle/packetcrypt_rs#8]
```
1619917946 WARN annmine.rs:432 [2137] handler [..] replied with no result [..]
1619917947 INFO poolclient.rs:125 Failed to make request Status code was 404 retry in 5 seconds
```

404 Error ist just "Info" not "Error" or "Warn" and does not tell the Pool/URL in https://github.com/cjdelisle/packetcrypt_rs/blob/02c00a9fefcd8ebbd5d39b559d27863bcfe2bd2e/packetcrypt-util/src/poolclient.rs#L75

Thanks!

**Comment by cjdelisle** [comment]
fixed in 28237abe5fef1aa06f86093aa0d87628e3d8507c

**Comment by Knight1** [comment]
Wow. That was fast. Thank you @cjdelisle !
